guides/source/getting_started.md

```diff
@@ -7,1150 +7,1554 @@ This guide covers getting up and running with Ruby on Rails.
-proper prerequisites installed. These include:
-
-* Ruby
-* SQLite3
-
-#### Installing Ruby
-
-Open up a command line prompt. On macOS open Terminal.app; on Windows choose
-"Run" from your Start menu and type `cmd.exe`. Any commands prefaced with a
-dollar sign `$` should be run in the command line. Verify that you have a
-current version of Ruby installed:
-
-```bash
-$ ruby --version
-ruby 3.2.0
-```
-
-Rails requires Ruby version 3.2.0 or later. It is preferred to use the latest Ruby version.
-If the version number returned is less than that number (such as 2.3.7, or 1.8.7),
-you'll need to install a fresh copy of Ruby.
+- **Don't Repeat Yourself:** DRY is a principle of software development which states that "Every piece of knowledge must have a single, unambiguous, authoritative representation within a system". By not writing the same information over and over again, our code is more maintainable, more extensible, and less buggy.
+- **Convention Over Configuration:** Rails has opinions about the best way to do many things in a web application, and defaults to this set of conventions, rather than require that you define them yourself through endless configuration files.
 
-To install Rails on Windows, you'll first need to install [Ruby Installer](https://rubyinstaller.org/).
+Creating a New Rails App
+------------------------
 
-For more installation methods for most Operating Systems take a look at
-[ruby-lang.org](https://www.ruby-lang.org/en/documentation/installation/).
+We're going to build a project called `store` - a simple e-commerce app that demonstrates several of Rails' built-in features.
 
-#### Installing SQLite3
+TIP: Any commands prefaced with a dollar sign `$` should be run in the terminal.
 
-You will also need an installation of the SQLite3 database.
-Many popular UNIX-like OSes ship with an acceptable version of SQLite3.
-Others can find installation instructions at the [SQLite3 website](https://www.sqlite.org).
+### Prerequisites
 
-Verify that it is correctly installed and in your load `PATH`:
+For this project, you will need:
 
-```bash
-$ sqlite3 --version
-```
-
-The program should report its version.
-
-#### Installing Rails
+* Ruby 3.2 or newer
+* Rails 8.0.0 or newer
+* A code editor
 
-To install Rails, use the `gem install` command provided by RubyGems:
+Follow the [Install Ruby on Rails Guide](install_ruby_on_rails.html) if you need to install Ruby and/or Rails.
 
-```bash
-$ gem install rails
-```
-
-To verify that you have everything installed correctly, you should be able to
-run the following in a new terminal:
+Let's verify the correct version of Rails is installed. To display the current version, open a terminal and run the following. You should see a version number printed out:
 
 ```bash
 $ rails --version
 Rails 8.1.0
 ```
 
-If it says something like "Rails 8.1.0", you are ready to continue.
+The version shown should be Rails 8.0.0 or higher.
```

Comment:
```suggestion
The version shown should be Rails 8.1.0 or higher.
```

Reply:
Keep 8.0.0 in all of those. We can change to 8.1.0 after it is released